src/app/components/common/node-editor/custom-node/custom-node.component.ts

```diff
@@ -412,6 +419,10 @@ export class CustomNodeComponent implements OnChanges {
     this.flowEditorService.removeFilter.next({ id: this.nodeId });
   }
 
+  showReadingsPerAsset() {
+    this.flowEditorService.showItemsInQuickview.next({ showPluginConfiguration: false, showFilterConfiguration: false, showLogs: false, showReadings: true, serviceName: this.service.name });
```

Comment:
In showItemsInQuickview  BehaviorSubject the only option that is required to open/show should be passed only. The condition checking on the quick view page can manage default value to false of other items.